src/components/Checkout/CheckoutReceipt.vue

```diff
@@ -329,6 +331,10 @@ export default {
 			type: Boolean,
 			default: false
 		},
+		enableKivaCardTracking: {
```

Comment:
@christian14b This prop should be renamed to something more generic, for example `enableButtonClickTracking`, since it also enables tracking for donation button.